src/dvclive/dvc.py

```diff
@@ -125,12 +128,23 @@ def _update_entries(old, new, key):
             del orig["artifacts"]
 
 
-def get_random_exp_name(scm, baseline_rev) -> str:
-    from dvc.repo.experiments.utils import (
-        get_random_exp_name as dvc_get_random_exp_name,
-    )
-
-    return dvc_get_random_exp_name(scm, baseline_rev)
+def get_exp_name(name, scm, baseline_rev) -> str:
+    from dvc.exceptions import InvalidArgumentError
+    from dvc.repo.experiments.refs import ExpRefInfo
+    from dvc.repo.experiments.utils import check_ref_format, get_random_exp_name
+
+    if name:
+        ref = ExpRefInfo(baseline_sha=baseline_rev, name=name)
+        if scm.get_ref(str(ref)):
+            logger.warning(f"Experiment conflicts with existing experiment '{name}'.")
```

Comment:
btw., what happens if two people name the experiment the same way simultaneously (feels that it would have been better to have still some random id / hash in the name or something).

Reply:
Nothing prevents us from adding a random suffix, but it's outside the scope of this PR, right? How about we open an issue in dvc to discuss it?

Reply:
yep, sorry, not a blocker for this PR